packages/workspace/test/workspace/hidden_values.test.ts

```diff
@@ -301,10 +301,14 @@ describe('handleHiddenChanges', () => {
         },
       })
       const { field } = objectType.fields
-      const change: DetailedChange = {
+      const change: DetailedChangeWithBaseChange = {
         id: field.elemID,
         action: 'add',
         data: { after: field },
+        baseChange: {
+          action: 'add',
+          data: { after: field },
+        },
```

Comment:
```suggestion
      const change = toDetailedChangeFromBaseChange(toChange({ after: field }))
```

same for other places in this file